qiskit_nature/algorithms/ground_state_solvers/ground_state_solver.py

```diff
@@ -58,6 +58,22 @@ def solve(
         """
         raise NotImplementedError
 
+    @abstractmethod
+    def get_qubit_operators(
+        self,
+        problem: BaseProblem,
+        aux_operators: Optional[ListOrDictType[Union[SecondQuantizedOp, PauliSumOp]]] = None,
+    ) -> Tuple[PauliSumOp, Optional[ListOrDictType[Union[SecondQuantizedOp, PauliSumOp]]]]:
+        """Prepare problem and ground_state_solver state for solve method.
```

Comment:
Needs to be updated to reflect that this is constructing the qubit operators by:
- getting the second quantized operators from the `problem` (potentially running a driver in doing so [can be computationally expensive])
- using a `QubitConverter` to map + reduce the operators to qubit ops